addons/website_sale/data/digest_kpi.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="kpi_website_sale_total" model="digest.kpi" forcecreate="True">
+        <field name="name">eCommerce Sales</field>
+        <field name="res_model_id" ref="model_sale_report"/>
+        <field name="agg_mode">sum</field>
+        <field name="agg_field">price_subtotal</field>
+        <field name="date_field">date</field>
+        <field name="domain">[('state', 'not in', ['draft', 'cancel', 'sent']), ('website_id', '!=', False)]</field>
+        <field name="group_ids" eval="[Command.link(ref('sales_team.group_sale_salesman_all_leads'))]"/>
+        <field name="action_id" ref="website.backend_dashboard"/>
+        <field name="action_menu_root_id" ref="website.menu_website_configuration"/>
+        <field name="module_id" ref="base.module_website_sale"/>
+        <field name="sequence" eval="12000"/>
+    </record>
+</odoo>
```

Comment:
Please pay attention to adding a new line at the end of the file. See https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
You can configure your IDE to automatically do it for you (`files.insertFinalNewline: true` in VSCode).
```suggestion
</odoo>

```